### DISCUSSION TITLE
Connecting Storage with Laravel via S3

### DISCUSSION THREAD

**Discussion by csoutham** [supabase/supabase#30518] [Questions]
Has anyone got a configuration working with Laravel and Supabase Storage via S3?

```
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=eu-central-1 <- region
AWS_ENDPOINT=https://_endpoint_.supabase.co/storage/v1/s3 <- I've tried using AWS_URL instead
AWS_BUCKET=images <- bucket name
AWS_USE_PATH_STYLE_ENDPOINT=false
```

Anything else that needs addressing? I can't connect with the above using Transmit (on the Mac) to test access either.

**Comment by rfgonzales** [comment]
were you able to figure this out? cant find anything in  supabase docs

**Reply by csoutham** [reply]
No @rfgonzales, the endpoint is read-only.
